### DISCUSSION TITLE
Interested in using GitHub sponsors for this plugin

### DISCUSSION THREAD

**Discussion by dfsnow** [folke/flash.nvim#171] [General]
Hey folke! Thank you for this _amazing_ plugin. It has quickly replaced basically all my other motion plugins and feels far more intuitive than everything else I've used. Great work!

If you're interested, I'd love to throw a few dollars per month your way in appreciation. I figure GitHub sponsors is probably the best way to do this, but it's not turned on. I briefly searched other repos and issues about GH Sponsors, but didn't dig anything up.

**Comment by folke** [comment]
Thank you for the kind words!
I'm not interested in sponsoring, but feel free to sponsor Neovim or a good cause instead.
